### PR TITLE
Add heroku access to log buckets

### PR DESCRIPTION
**Pull request by danlamanna**
Related https://github.com/dandi/dandi-archive/issues/1424

### CODE REVIEW

**Comment by danlamanna** [comment]
@satra @yarikoptic When looking into this PR I realized master isn't building cleanly anymore, it looks like a regular `terraform plan` wants to make a lot of [changes](https://app.terraform.io/app/dandi/workspaces/dandi-prod/runs/run-GqLJPLsqffQKBkuY) to S3 buckets. Is it possible manual changes were made through the GUI recently?

**Comment by yarikoptic** [comment]
I did no changes in GUI any time not only recently.

**Comment by satra** [comment]
neither did i.

File: terraform/modules/dandiset_bucket/main.tf [hunk 1]
```diff
@@ -90,6 +92,33 @@ resource "aws_s3_bucket" "log_bucket" {
...
 
+data "aws_iam_policy_document" "dandiset_log_bucket_policy" {
+  statement {
+    resources = [
+      "${aws_s3_bucket.log_bucket.arn}",
+      "${aws_s3_bucket.log_bucket.arn}/*",
+    ]
+
+    actions = [
+      # Needed for the app to process logs for collecting download analytics
```
Comment (danlamanna) [inline terraform/modules/dandiset_bucket/main.tf:103]: @mvandenburgh I want to start loosely documenting which features need permissions for future readers of the Terraform.

**Review by mvandenburgh — APPROVED** [review]

**Pull request merged**